app/models/patron.rb

```diff
@@ -2,10 +2,6 @@
 
 # Class to model Patron information
 class Patron
-  include ActiveModel::Validations
-  include ActiveModel::Conversion
-  extend ActiveModel::Naming
-
```

Comment:
don't we need these for the patron validation check on key?

Reply:
yes, this was a mistake, should be corrected now